.travis.yml

```diff
@@ -31,6 +31,7 @@ before_script:
   - sudo redis-server /etc/redis/redis.conf --port 6380 --requirepass 'my-secret'
   - bash contracts/compile.sh
 script:
+  - truffle version
```

Comment:
@sarvesh-ost it's not needed

Reply:
Yes, I will remove this. It was added to check truffle and solc version on Travis.